sdk/python/tests/test.py

```diff
@@ -32,35 +36,30 @@ def test_log_metadata(self):
             data_set_id="123",
             model_id="12345",
             metrics_type=metadata.Metrics.VALIDATION,
-            values={
-                "accuracy": 0.95},
-            labels={
-                "mylabel": "l1"}))
+            values={"accuracy": 0.95},
+            labels={"mylabel": "l1"}))
+    print(metrics)
+
+    model = r.log(
+        metadata.Model(
+            name="MNIST",
+            description="model to recognize handwritten digits",
+            owner="someone@kubeflow.org",
+            uri="gcs://my-bucket/mnist",
+            model_type="neural network",
+            training_framework={
+                "name": "tensorflow",
+                "version": "v1.0"
+            },
+            hyperparameters={
+                "learning_rate": 0.5,
+                "layers": [10, 3, 1],
+                "early_stop": True
+            },
+            version="v0.0.1",
+            labels={"mylabel": "l1"}))
+    print(model)
```

Comment:
Maybe we can add validation here instead of print? WDYT?

Reply:
Good suggestion. I added assertions to verify the data is inserted.